Addressbook/src/module/delete-user.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/ash
+
+. module/search-user.sh
+
+db="../database/addressbook.db"
```

Comment:
It is a nice touch that the path is stored in a variable :+1: 
However, it still appears multiple times at the top of the modules, so it would be best to move it out to it's own `module/config.sh`.